xds/src/main/java/com/linecorp/armeria/xds/XdsClusterManager.java

```diff
@@ -48,11 +50,14 @@ final class XdsClusterManager implements SafeCloseable {
     private final EventExecutor eventLoop;
     private final Bootstrap bootstrap;
 
-    XdsClusterManager(EventExecutor eventLoop, Bootstrap bootstrap) {
+    XdsClusterManager(EventExecutor eventLoop, Bootstrap bootstrap, MeterIdPrefix meterIdPrefix,
+                      MeterRegistry meterRegistry) {
         this.eventLoop = eventLoop;
         this.bootstrap = bootstrap;
         localClusterName = bootstrap.getClusterManager().getLocalClusterName();
         if (!Strings.isNullOrEmpty(localClusterName) && bootstrap.getNode().hasLocality()) {
+            final DefaultXdsLoadBalancerLifecycleObserver observer =
+                    new DefaultXdsLoadBalancerLifecycleObserver(meterIdPrefix, meterRegistry, localClusterName);
```

Comment:
I understand that there’s still no extension point for users to configure a custom XdsLoadBalancerLifecycleObserver.